cmd/fs-v1-multipart.go

```diff
@@ -361,9 +361,9 @@ func (fs fsObjects) PutObjectPart(bucket, object, uploadID string, partID int, s
 		return "", toObjectErr(err, minioMetaMultipartBucket, uploadIDPath)
 	}
 
+	// Append the part in background.
+	errCh := fs.bgAppend.append(fs.storage, bucket, object, uploadID, fsMeta)
```

Comment:
It looks like [when](https://github.com/minio/minio/pull/3419/files#diff-835acf3d529848c880629f8de34156ecR156) ``appendNeeded`` is true and ``appendPart`` function returned ``nil`` nothing is sent on ``errCh``.